src/components/AuthForm.js

```diff
@@ -14,16 +14,26 @@ const anonymousAuthData = server => ({
 });
 const KNOWN_AUTH_METHODS = ["basicauth", "account", "fxa", "ldap", "portier"];
 
+function debounce(fn, delay) {
+  var timer = null;
+  return function() {
+    var context = this,
+      args = arguments;
+    clearTimeout(timer);
+    timer = setTimeout(() => fn.apply(context, args), delay);
+  };
+}
```

Comment:
**nitpick**:
I have the impression that you could remove the context closure with a short array function no? (since this is maintained)
```js
function debounce(fn, delay) {
  var timer = null;
  return (...args) => {
    clearTimeout(timer);
    timer = setTimeout(() => fn.apply(this, args), delay);
  };
}
```

Reply:
I believe this would work, indeed, I'll give it a try.

You mean
```javascript
timer = setTimeout(() => fn(args), delay);
```

Right?